addons/account_payment/static/src/js/payment_form.js

```diff
@@ -0,0 +1,18 @@
+/** @odoo-module **/
+
+import PaymentForm from '@payment/js/payment_form';
+PaymentForm.include({
+    _prepareTransactionRouteParams() {
+        let invoice_ids = this.paymentContext['invoiceIds'];
+        let payment_reference = this.paymentContext['paymentReference'];
+        let payment_transaction_params = {};
+        if (invoice_ids) {
```

Comment:
For me those variable are a bit useless but if you think it adds clarity then ok :smile: